src/main/java/seedu/address/logic/commands/queue/ResumeCommand.java

```diff
@@ -17,13 +17,14 @@
     public static final String COMMAND_WORD = "resume";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Doctor is back from his/her break\n"
+            + ": Doctor is back from his/her break. Allows patients to be directed to the doctor "
+            + "based on the room number.\n"
```

Comment:
Please remove this line. No reference to room numbers please.

Reply:
Removed.